src/py2spack/conversion_tools.py

```diff
@@ -64,19 +62,7 @@ def requirement(self) -> str | None:
 
 
 # TODO @davhofer: integrate this into a datastructure, omit mutable global state  # noqa: TD003
-evalled: dict = {}
-
-
-def acceptable_version(version: str) -> pv.Version | None:
```

Comment:
Renamed (`_parse_packaging_version`) and moved method to `package_providers` module